### ISSUE TITLE
Add "extract audio" option to reduce the size of the downloaded file and automatically extract the audio track from dailymotion/facebook videos

### ISSUE THREAD

**Issue by trimechee** [persepolisdm/persepolis#980]
Hello, when we sleep we often listen to rare documentaries in French on crime which only exist on the dailymotion site, for example:

https://www.dailymotion.com/video/x8xx4py

and to save disk space ot quicly transfer the audio files to our samrtphone via snapdrop website for exemple, it will be great to automatically extract the audio track with this automatic audio extraction preset for dailymotion and facebook videos :

-f bestaudio/worst

or

-f bestaudio/worst -x

Thank you so much !

Explanation :
With the "bestaudio" preset, according to my test, very fast and smooth downloading no problem, perfect 128 kb/s audio quality, with the "worsaudio" preset, very slow downloading and I was used to using yt-dlp with aria2c to speed up the download with the worstaudio preset, in addition with the "worstaudio" preset, it is impossible to move in the audio file hence the importance of adding "-x" i.e. the perst worst audio must be in the form " -f bestaudio -x", the problem I often use the "bestaudio" preset, but neither dailymotion nor facebook are compatible with this bestaudio preset and I have to delete this preset " bestaudio" when I download dailymotion and facebook videos and reactivate this "bestaudio" preset only with youtube videos which is tiring and boring, so our dream please is to have a universal preset that works with all sites and the preset" -f bestaudio/worst -x" seems to fix this problem and downloads audio from all sites and seems to convert the dailymotion video to an "m4a" audio file in some cases according to my tests like for this video:

https://www.dailymotion.com/video/x8mdd2c

....so my request please is to add an optional preset "-f bestaudio/worst -x" or "-f bestaudio/worst" which works with all sites, thank you very much!

**Comment by trimechee** [comment]
Explanation :
With the "bestaudio" preset, according to my test, very fast and smooth downloading no problem, perfect 128 kb/s audio quality, with the "worsaudio" preset, very slow downloading and I was used to using yt-dlp with aria2c to speed up the download with the worstaudio preset, in addition with the "worstaudio" preset, it is impossible to move in the audio file hence the importance of adding "-x" i.e. the perst worst audio must be in the form " -f bestaudio -x", the problem I often use the "bestaudio" preset, but neither dailymotion nor facebook are compatible with this bestaudio preset and I have to delete this preset " bestaudio" when I download dailymotion and facebook videos and reactivate this "bestaudio" preset only with youtube videos which is tiring and boring, so our dream please is to have a universal preset that works with all sites and the preset" -f bestaudio/worst -x" seems to fix this problem and downloads audio from all sites and seems to convert the dailymotion video to an "m4a" audio file in some cases according to my tests like for this video:

https://www.dailymotion.com/video/x8mdd2c

....so my request please is to add an optional preset "-f bestaudio/worst -x" or "-f bestaudio/worst" which works with all sites, thank you very much!

**Comment by alireza-amirsamimi** [comment]
Hello
Unfortunately, unlike YouTube, it is not possible to download audio from Daily Motion separately.

**Comment by trimechee** [comment]
Yes, that's why the preset worst -x to extract audio from the worst video quality , we hope it will work with dailymotion, facebook and thousands  of websites supported by yt-dlp

**Comment by alireza-amirsamimi** [comment]
Persepolis uses [yt-dlp](https://github.com/yt-dlp/) library to find video and audio. You must ask that project for this feature request.

**Comment by trimechee** [comment]
Yes, this preset is tested by another developer of download manager and by me and it is the best preset ever without bugs to extract audio from videos :)
